### DISCUSSION TITLE
Create notifications in batch

### DISCUSSION THREAD

**Discussion by avcpu** [magicbell/community#54] [Questions] [answered]
Can notifications be created in batch via REST API? I want to reduce number of requests, so checking how it can be done.

**Comment by unamashana** [comment, marked as answer]
@avcpu you can specify multiple recipients in one API call and customize the content (if needed) using the recipient's data (including their `custom_attributes`). Please see the docs for the syntax for specifying recipients - https://www.magicbell.com/docs/rest-api/reference#notifications-create. 

Happy to expand further if you have any questions.

**Reply by avcpu** [reply]
Thank you for quick answer. It's convenient to have multiple recipients in one notification object, but maybe there is any possibility (or future plans) to send one request with array of notification objects? In such way it will be possible to create several notifications with differnent properties in one request.

**Reply by smeijer** [reply]
Hi @avcpu , I know some time has passed, but can you clarify what you mean by "different properties"? What part of the notification would you like to customise per recipient?
